apps/strapi-admin-extensions/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "declaration": true,
+    "module": "commonjs",
+    "outDir": "dist",
+    "lib": ["ES2021", "DOM"],
+    "strict": true
+  },
+  "include": ["src/**/*", "./jest.config.ts"],
+  "exclude": ["node_modules", "**/*.test.ts", "./src/types/openapi.ts"]
+}
```

Comment:
Zou het handig zijn om deze een base `tsconfig.json` te laten extenden? Dan kunnen we de settings meer in sync houden, en hoeven we minder configuratie files  te onderhouden.